scripts/install.sh

```diff
@@ -81,30 +76,68 @@ After=network-online.target
 
 [Service]
 ExecStart=$BINDIR/ollama serve
-User=ollama
-Group=ollama
+User=$(whoami)
+Group=$(id -gn)
 Restart=always
 RestartSec=3
-Environment="HOME=/usr/share/ollama"
+Environment="HOME=$(eval echo ~$(whoami))"
```

Comment:
A user service is problematic because the service serves on TCP. Anyone with access to the address/port will be able to send requests to the server and impersonate that user.